usr/lib/live-installer/installer.py

```diff
@@ -416,9 +416,17 @@ def finish_install(self, setup):
             if(line.startswith("XKBMODEL=")):
                 newconsolefh.write("XKBMODEL=\"%s\"\n" % setup.keyboard_model)
             elif(line.startswith("XKBLAYOUT=")):
+                if setup.keyboard_layout in ('am', 'ara', 'ben', 'bd', 'bg', 'bt', 'by', 'deva', 'ge',
+                      'gh', 'gr', 'guj', 'guru', 'il', 'in', 'ir', 'iku',
+                      'kan', 'kh', 'kz', 'la', 'lao', 'lk', 'mk', 'mm', 'mn',
+                      'mv', 'mal', 'ori', 'pk', 'ru', 'scc', 'sy', 'syr',
+                      'tel', 'th', 'tj', 'tam', 'ua', 'uz'):
```

Comment:
Define the list somewhere separately:

```py
NONLATIN_KEYBOARD_LAYOUTS = ...
```
Will also make it defined in a single place and can be reused (e.g. as in https://github.com/linuxmint/live-installer/pull/89).